scripts/postgres/restoreDbFromSource.ts

```diff
@@ -215,8 +215,61 @@ async function restoreFromBackup({
   });
   const targetPassword = await targetSigner.getAuthToken();
 
+  // pg_restore only drops tables in the dump, so we drop all tables before pg_restore.
```

Comment:
This script file is getting long/clumsy. I will see if there are easy ways to simplify.